portal/models.py

```diff
@@ -72,6 +80,7 @@ class Host(models.Model):
     name = models.CharField(max_length=242, unique=True,
         validators=[domain_validator])
     type = models.CharField(max_length=24, choices=HOST_TYPES)
+    os = models.CharField(max_length=32, choices=OS_TYPES)
```

Comment:
Please use `null=True` so that we don't need to identify all of the existing hosts in the migration. This will allow OS to be NULL until you try to edit the host in the admin interface.

Reply:
Done.